src/dialogs/InvalidFilingDialog.vue

```diff
@@ -1,11 +1,11 @@
 <template>
```

Comment:
I generalized this error dialog so that it can be used for:
- invalid dissolution
- invalid restoration
- invalid incorporation application